composer.json

```diff
@@ -78,7 +78,7 @@
         "symfony/phpunit-bridge": "^3.3 || ^4.0"
     },
     "suggest": {
-        "liip/imagine-bundle": "^0.9",
+        "liip/imagine-bundle": "If you want on-the-fly thumbnail generation or image filtering (scale, crop, watermark...)",
```

Comment:
Shouldn't we add liip/imagine-bundle to require-dev and conflict section? Just to be sure people install the required version and we tests against it too.